xmipp

```diff
@@ -655,18 +684,50 @@ def checkCompiler(configDict):
 
 def configMPI(configDict):
     mpiDetected = []
-    if configDict["MPI_CC"] == "" and checkProgram("mpicc"):
-        configDict["MPI_CC"]="mpicc"
-        mpiDetected.append('mpicc')
-    if configDict["MPI_CXX"] == "" and checkProgram("mpiCC"):
-        configDict["MPI_CXX"]="mpiCC"
-        mpiDetected.append('mpiCC')
+    if configDict["MPI_RUN"] == "":
+        if checkProgram("mpirun", False):
+            configDict["MPI_RUN"] = "mpirun"
+            mpiDetected.append('mpirun')
+        elif checkProgram("mpiexec", False):
+            configDict["MPI_RUN"] = "mpiexec"
+            mpiDetected.append('mpiexec')
+        else:
+            mpiDir = findFileInDirList('mpirun', ['/usr/lib/openmpi/bin',
+                                                  '/usr/lib64/openmpi/bin'])  # add candidates if not found
+            if mpiDir:
+                configDict["MPI_RUN"] = mpiDir + "/mpirun"
+                updateXmippEnv(PATH=mpiDir)
+                mpiDetected.append(mpiDir + "/mpirun")
+    if configDict["MPI_CC"] == "":
+        if checkProgram("mpicc", False):
+            configDict["MPI_CC"] = "mpicc"
+            mpiDetected.append('mpicc')
+        else:
+            mpiDir = findFileInDirList('mpicc', ['/usr/lib/openmpi/bin',
+                                                 '/usr/lib64/openmpi/bin'])  # add candidates if not found
+            if mpiDir:
+                configDict["MPI_CC"] = mpiDir + "/mpicc"
+                mpiDetected.append(mpiDir + "/mpicc")
+            else:
+                checkProgram("mpicc")  # To raise the hint
+    if configDict["MPI_CXX"] == "":
+        if checkProgram("mpiCC", False):
+            configDict["MPI_CXX"] = "mpiCC"
```

Comment:
Yes. It is like before. The mpi compiler agains C++ code is mpiCC.

Reply:
See https://www.open-mpi.org/doc/v3.0/man1/mpicxx.1.php and https://www.open-mpi.org/doc/v3.0/man1/mpicc.1.php

Reply:
I see, I don't know too much about this, but I see in Ubuntu both are the same:

```
$ mpiCC -v
Using built-in specs.
COLLECT_GCC=/usr/bin/g++
...

$ mpicxx -v
Using built-in specs.
COLLECT_GCC=/usr/bin/g++
...

$ mpicc -v
Using built-in specs.
COLLECT_GCC=/usr/bin/gcc
...
```

But right, I find much more documentation about `mpicxx` rather than about `mpiCC`...